src/src/common.c

```diff
@@ -8,9 +8,8 @@
 
 uint8_t rxPacket[16];
 uint8_t txPacket[18];
-uint8_t vtxModeLocked;
+uint8_t vtxModeLocked = 0;
```

Comment:
No need to init zero. That is done automatically at start up phase (check bss section :) ).
No need to fix now!

Reply:
I knew you wouldn't like this LOL

At the time I did it for my own readability.  Ill revert it.